.goreleaser.yaml

```diff
@@ -34,10 +34,6 @@ changelog:
       - '^test:'
 release:
   name_template: "{{.ProjectName}}-v{{.Version}}"
-  header: |
```

Comment:
Why is this removed? It's used to generate the GitHub release.

Reply:
I thought that this was coming from boilerplate and removed it to make desc a bit shorter. Do you want to keep that?

Reply:
Sure, done!